src/configuration.js

```diff
@@ -80,58 +54,58 @@ export default class Configuration {
         }
 
         if (type === 'remote') {
```

Comment:
if at this point you have not created a sampler and the `type` is not empty and not `remote`, then the `type` is unsupported value, thus exception. 
```
if (!sampler) {
  if (type && type != 'remote') {
    throw
  }
  sampler = new RemoteSampler(serviceName, {
                  sampler: new ProbabilisticSampler(param)
  });  
}
```